pack/tsk/tskp.json

```
@@ -584,6 +657,24 @@
         "type_code": "asset",
         "xp": 0
     },
+    {
+        "code": "09086",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "flavor": "By both light and shadow / may dusk beget dawn; May the turning of the world / lead to power everlasting.",
```

Should we add line breaks to this, to represent the formatting of the poem? 
```suggestion
        "flavor": "By both light and shadow /\nmay dusk beget dawn;\nMay the turning of the world /\nlead to power everlasting.",
```

Maybe too ott?

---

We wouldn't usually, but I wondered because it's formatted like a poem...